packages/frontend/src/hooks/echarts/useEcharts.ts

```diff
@@ -66,6 +66,46 @@ const getAxisTypeFromField = (item?: Field): string => {
     }
 };
 
+export const getAxisDefaultMinValue = ({
```

Comment:
Could these be simplified into a function that takes the existing min, max and returns a new one?

I also wonder if the logic inside could be made more explicit:

```
switch(Math.sign(min) + Math.sign(max)) {
  case: 2
    # both are positive - adjust min value
  case: 0
    # data is both sides of zero - do nothing
  case: -2
    # both are negative - adjust max value